### PR TITLE
Updated TF Serving Dockerfiles to support TensorFlow 2.9.1

### PR DESCRIPTION
**Pull request by srihari-humbarwadi-intel**

### CODE REVIEW

File: docker/ubuntu18/openvino_tensorflow_cgvh_runtime_2.1.0-serving.dockerfile [hunk 1]
```diff
@@ -1,69 +1,20 @@
...
-# Run Unit Tests
-RUN source build_cmake/venv-tf-py3/bin/activate && \
-    source ${INTEL_OPENVINO_DIR}/setupvars.sh && \
-    python3 test_ovtf.py
+#######################################################################################
+FROM openvino/openvino_tensorflow_ubuntu18_runtime:${OVTF_VERSION} as ovtf_runtime
+#######################################################################################
 
-CMD ["/bin/bash"]
+LABEL description="This is the runtime image for OpenVINO™ integration with TensorFlow on Ubuntu 18.04 LTS"
```
Comment (sspintel) [inline docker/ubuntu18/openvino_tensorflow_cgvh_runtime_2.1.0-serving.dockerfile:13]: We can remove this LABEL description

File: docker/ubuntu18/openvino_tensorflow_cgvh_runtime_2.1.0-serving.dockerfile [hunk 2]
```diff
@@ -1,69 +1,20 @@
...
+LABEL vendor="Intel Corporation"
```
Comment (sspintel) [inline docker/ubuntu18/openvino_tensorflow_cgvh_runtime_2.1.0-serving.dockerfile:14]: Move this LABEL vendor to Line No 21

File: docker/ubuntu20/openvino_tensorflow_cgvh_runtime_2.1.0-serving.dockerfile [hunk 3]
```diff
@@ -106,12 +60,14 @@ RUN bazel clean --expunge --color=yes && \
...
 
 ################################################################################
-FROM openvino/openvino_tensorflow_ubuntu20_runtime:2.0.0 AS runtime_serving
+FROM ovtf_runtime AS ovtf_serving_runtime
 ################################################################################
 
 LABEL description="This is the TF Serving runtime image for OpenVINO™ integration with TensorFlow on Ubuntu 20.04 LTS"
 LABEL vendor="Intel Corporation"
 
+ARG TF_VERSION
```
Comment (sspintel) [inline docker/ubuntu20/openvino_tensorflow_cgvh_runtime_2.1.0-serving.dockerfile:69]: Why is this ARG required here?

Comment (sspintel) [reply docker/ubuntu20/openvino_tensorflow_cgvh_runtime_2.1.0-serving.dockerfile:69]: The TF_VERSION ARG at the beginning of the file still should work, right?

File: docker/ubuntu18/openvino_tensorflow_cgvh_runtime_2.1.0-serving.dockerfile (hunk 1 above)
Comment (srihari-humbarwadi-intel) [reply docker/ubuntu18/openvino_tensorflow_cgvh_runtime_2.1.0-serving.dockerfile]: done

**Pull request merged**